### ISSUE TITLE
Add support to django 3

### ISSUE THREAD

**Issue by zetaab** [zyegfryed/django-statici18n#47]
Currently this library does not support django 3:

```
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/myuser/.virtualenvs/myenv/lib/python3.6/site-packages/django/utils/autoreload.py", line 53, in wrapper
    fn(*args, **kwargs)
  File "/Users/myuser/.virtualenvs/myenv/lib/python3.6/site-packages/django/core/management/commands/runserver.py", line 109, in inner_run
    autoreload.raise_last_exception()
  File "/Users/myuser/.virtualenvs/myenv/lib/python3.6/site-packages/django/utils/autoreload.py", line 76, in raise_last_exception
    raise _exception[1]
  File "/Users/myuser/.virtualenvs/myenv/lib/python3.6/site-packages/django/core/management/__init__.py", line 357, in execute
    autoreload.check_errors(django.setup)()
  File "/Users/myuser/.virtualenvs/myenv/lib/python3.6/site-packages/django/utils/autoreload.py", line 53, in wrapper
    fn(*args, **kwargs)
  File "/Users/myuser/.virtualenvs/myenv/lib/python3.6/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/myuser/.virtualenvs/myenv/lib/python3.6/site-packages/django/apps/registry.py", line 122, in populate
    app_config.ready()
  File "/Users/myuser/.virtualenvs/myenv/lib/python3.6/site-packages/statici18n/apps.py", line 8, in ready
    from . import conf  # noqa
  File "/Users/myuser/.virtualenvs/myenv/lib/python3.6/site-packages/statici18n/conf.py", line 3, in <module>
    from appconf import AppConf
  File "/Users/myuser/.virtualenvs/myenv/lib/python3.6/site-packages/appconf/__init__.py", line 2, in <module>
    from .base import AppConf  # noqa
  File "/Users/myuser/.virtualenvs/myenv/lib/python3.6/site-packages/appconf/base.py", line 4, in <module>
    from django.utils import six
ImportError: cannot import name 'six'
```

**Comment by zetaab** [comment]
https://github.com/zyegfryed/django-statici18n/blob/master/requirements.txt#L1 this needs to be updated to 1.0.3

**Comment by zyegfryed** [comment]
@zetaab I released [v1.9.0](https://django-statici18n.readthedocs.io/en/latest/changelog.html#v1-9-0-2020-jan-11) which brings Django 3 support. Thanks for your feedback!
